engine/src/main/scala/cromwell/engine/EngineELF.scala

```diff
@@ -0,0 +1,29 @@
+package cromwell.engine
+
+import java.nio.file.{FileSystem, FileSystems}
+
+import cromwell.backend.wdl.{PureFunctions, ReadLikeFunctions}
+import cromwell.core.WorkflowOptions
+import wdl4s.expression.WdlStandardLibraryFunctions
+import wdl4s.values.{WdlFile, WdlValue}
+
+import scala.util.{Failure, Try}
+
+class EngineELF(workflowOptions: WorkflowOptions) extends WdlStandardLibraryFunctions with ReadLikeFunctions with PureFunctions {
```

Comment:
Yeah me too :/ I was trying to be consistent.
It's easy to rename anyway, maybe we could take a poll on what they should be renamed to ? :p